c/validation/adbc_validation_statement.cc

```diff
@@ -2180,15 +2180,15 @@ void StatementTest::TestSqlBind() {
 
   ASSERT_THAT(
       AdbcStatementSetSqlQuery(
-          &statement, "SELECT * FROM bindtest ORDER BY \"col1\" ASC NULLS FIRST", &error),
+          &statement, "SELECT * FROM bindtest ORDER BY col1 ASC NULLS FIRST", &error),
```

Comment:
It's more about consistency. Our `CREATE TABLE` query earlier in this function doesn't quote the column names, so our select statement needs to also *not* quote the names. Almost everywhere else we quote the columns. We just need to be consistent.

That said, I agree with it would be awesome for these tests to be more data-driven.